frontend/src/App.js

```diff
@@ -2,15 +2,20 @@ import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 import HomePage from "./pages/HomePage";
 import Recipes from "./pages/Recipes";
 import useIngredients from "./hooks/useIngredients";
+import InitialIngredients from "./hooks/InitialIngredrients";
```

Comment:
By convention hooks are names `useSomething`

Reply:
Will rename it.